File: sorting/js/bubbleSort.js
let arr = [12, 34, 67, 1, 90, 78]

for(let i = 0; i < arr.length - 1; i++) {
    for(let j = 0; j <arr.length - i - 1; j++) {
        if(arr[j] > arr[j + 1]) {
            let temp = arr[j];
            arr[j] = arr[j + 1];
            arr[j + 1] = temp;
        }
    }
}

console.log(arr);

File: sorting/java/BubbleSort.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class BubbleSort {

    public static void main(String[] args) {

        List<Integer> arr = new ArrayList<>(Arrays.asList(1, 9, 2, 0, 4, 3));

        bubbleSort(arr);

        System.out.println(arr);

    }

    public static void bubbleSort(List<Integer> arr) {
        for (int i = 0; i < arr.size() - i; i++) {
            for (int j = 0; j < arr.size() - i - 1; j++) {
                if (arr.get(j) > arr.get(j + 1)) {
                    int temp = arr.get(j);
                    arr.set(j, arr.get(j + 1));
                    arr.set(j + 1, temp);
                }
            }
        }
    }


}
